pkg/kubectl/cmd/drain.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/jonboulle/clockwork"
 	"github.com/spf13/cobra"
 
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Yes we already have that. And depending on `k8s.io/api` is considered `good` dependency.